src/devices/Ssd13xx/Ssd1306.cs

```diff
@@ -2,48 +2,44 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Iot.Device.Ssd1306.Command;
+using Iot.Device.Ssd13xx.Commands;
 using System;
 using System.Device.I2c;
 
-namespace Iot.Device.Ssd1306
+namespace Iot.Device.Ssd13xx
 {
     /// <summary>
     /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
     /// light emitting diode dot-matrix graphic display system. 
     /// </summary>
-    public class Ssd1306 : IDisposable
+    public class Ssd1306 : Ssd13xx
     {
-        private I2cDevice _i2cDevice;
-
         /// <summary>
         /// Initializes new instance of Ssd1306 device that will communicate using I2C bus.
         /// A single-chip CMOS OLED/PLED driver with controller for organic/polymer
         /// light emitting diode dot-matrix graphic display system. 
         /// </summary>
         /// <param name="i2cDevice">>The I2C device used for communication.</param>
         public Ssd1306(I2cDevice i2cDevice)
+        : base(i2cDevice)
         {
-            _i2cDevice = i2cDevice;
         }
 
-        /// <summary>
-        /// Verifies value is within a specific range.
-        /// </summary>
-        /// <param name="value">Value to check.</param>
-        /// <param name="start">Starting value of range.</param>
-        /// <param name="end">Ending value of range.</param>
-        /// <returns>Determines if value is within range.</returns>
-        internal static bool InRange(uint value, uint start, uint end)
+        public void SendCommand(ISsd1306Command command)
+        {
+            SendCommand((ICommand)command);
```

Comment:
nit: is the cast needed?

Reply:
Yep. This is needed in this particular place to call ICommand overload.